### PR TITLE
handyPrintPro: submission

### PR DESCRIPTION
**Pull request by i0ntempest**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --new-cask <cask>` worked successfully.
- [x] `brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

**Comment by i0ntempest** [comment]
Someone please help fixing the live check stanza, I have no experience with regex

**Comment by neutric** [comment]
> Someone please help fixing the livecheck stanza, I have no experience with regex

`regex(%r{href=.*?/handyPrintPRO\+v?(\d+(?:\.\d+)+)\.dmg}i)`

The `+` is a quantifier and has special meaning in regular expressions. Escaping with `\` should help.

File: Casks/handyprintpro.rb [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+  url "https://s3.amazonaws.com/netputing/handyPrintPRO/handyPrintPRO+v#{version}.dmg",
+      verified: "s3.amazonaws.com/netputing/handyPrintPRO/"
+  name "handyPrintPRO"
+  desc "AirPrint server"
+  homepage "http://www.netputing.com/handyprintpro/"
+
+  livecheck do
+    url :homepage
+    strategy :page_match
```
Comment (neutric) [inline Casks/handyprintpro.rb:13]: `strategy :page_match` is the default in this case and can thus be omitted.

Comment (i0ntempest) [reply Casks/handyprintpro.rb]: Done, thank you

**Comment by neutric** [comment]
Just now noticed: the current release was over three years ago, which means it'll most likely not be accepted as it is [considered unmaintained](https://docs.brew.sh/Acceptable-Casks#rejected-casks):

> The Cask is for an unmaintained app (no releases in the last year, or explicitly discontinued).

**Comment by miccal** [comment]
Thank you @i0ntempest, but as pointed out by @neutric this seems unmaintained.

**Comment by i0ntempest** [comment]
I didn’t find discontinue notice anywhere on their site and they’re still issuing license codes (I bought a copy 2 days ago), if that helps…

File: Casks/handyprintpro.rb [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  uninstall launchctl: "com.netputing.handyPrintProDaemon",
+            quit:      "com.netputing.handyPrintPro",
+            pkgutil:   [
+              "com.netputing.handyPrintPro",
+              "com.netputing.handyPrintProDaemon",
+              "com.netputing.handyPrintProLaunchd",
+            ]
+
+  zap trash: [
+    "/Library/Application Support/handyPrintPro",
```
Comment (miccal) [inline Casks/handyprintpro.rb:27]: Are you sure this is not meant to be `"~/Library/Application Support/handyPrintPro",`?

Comment (i0ntempest) [reply Casks/handyprintpro.rb:27]: Yes it's the global Library folder
image

**Comment by i0ntempest** [comment]
I'm getting HTTP error 500 when creating new PR here, what's up with that?

**Comment by miccal** [comment]
> I'm getting HTTP error 500 when creating new PR here, what's up with that?

Not sure, but I have reported it.

**Comment by neutric** [comment]
> > I'm getting HTTP error 500 when creating new PR here, what's up with that?
> 
> Not sure, but I have reported it.

Where do you report that? I have that problem every few days for the last few weeks. It usually lasts several hours.

**Comment by bevanjkay** [comment]
@neutric Just to GitHub support

**Comment by neutric** [comment]
Roger that!

**Pull request merged**